docs/dev/Lua API.rst

```diff
@@ -6222,6 +6229,28 @@ rendering on the next line(s) if all tabs cannot fit on a single line.
            itself as the second. The default implementation, which will handle most
            situations, returns ``self.active_tab_pens``, if ``self.get_cur_page() == idx``,
            otherwise returns ``self.inactive_tab_pens``.
+:scroll_key: Specifies a keybinding that can be used to scroll the tabs to the right.
+             Defaults to ``CUSTOM_ALT_T``.
+:scroll_key_back: Specifies a keybinding that can be used to scroll the tabs to the left.
+                  Defaults to ``CUSTOM_ALT_Y``.
+:scroll_left_text: The text to display on the left scroll label.
+                   Defaults to "<<<".
+:scroll_right_text: The text to display on the right scroll label.
+                    Defaults to ">>>".
```

Comment:
As a matter of style consistency, I don't think this should be configurable. I think players would expect this to be constant in all situations where it appears.

I also suggest making the horizontal scroll icon look more like this:
![image](https://github.com/user-attachments/assets/8e7df521-f9c8-4456-a03d-73eb486f53d5)

perhaps with only one left/right triangle  if two triangles stacked vertically doesn't fit well. The code for that example screen is here: 
- widget: https://github.com/DFHack/dfhack/blob/develop/plugins/lua/sort.lua#L1144
- characters: https://github.com/DFHack/dfhack/blob/develop/plugins/lua/sort.lua#L1015

Reply:
Ah yeah that's a much nicer symbol. Will fix!